manifests/base/mc-fleet-deployment.yaml

```
@@ -16,10 +16,16 @@ spec:
       subPath: konfigure
   initContainers:
   - args:
-    - --vault-address={{ .Values.Installation.V1.Auth.Vault.Address }}
+    - --vault-address=$(VAULT_ADDR)
     - --vault-role=argo
     - --vault-token-secret-name=argocd-vault-token
     - --vault-token-secret-namespace=argocd
+    env:
+    - name: VAULT_ADDR
+      valueFrom:
+        configMapKeyRef:
+          name: vault-address
+          key: vault-address
```

My suggestion so it's aligned with the default vault env var:

```suggestion
          key: VAULT_ADDR
```